README.md

```diff
@@ -26,261 +27,17 @@ This Terraform provider enables users to manage [Checkly](https://checklyhq.com)
-  name                      = "Example check"
-  type                      = "BROWSER"
-  activated                 = true
-  should_fail               = false
-  frequency                 = 10
-  double_check              = true
-  ssl_check                 = true
-  use_global_alert_settings = true
-  locations = [
-    "us-west-1"
-  ]
-
-  script = <<EOT
-const assert = require("chai").assert;
-const puppeteer = require("puppeteer");
-
-const browser = await puppeteer.launch();
-const page = await browser.newPage();
-await page.goto("https://google.com/");
-const title = await page.title();
-
-assert.equal(title, "Google");
-await browser.close();
-
-EOT
-}
-```
-
-An alternative syntax for add the script is by referencing an external file
-
-```terraform
-data "local_file" "browser-script" {
-  filename = "${path.module}/browser-script.js"
-}
-
-resource "checkly_check" "browser-check-1" {
-  ...
-  script = data.local_file.browser-script.content
-}
-```
-
-### Check Groups
-
-Checkly's groups feature allows you to group together a set of related checks, which can also share default settings for various attributes. Here is an example check group:
-
-```terraform
-resource "checkly_check_group" "test-group1" {
-  name      = "My test group 1"
-  activated = true
-  muted     = false
-  tags = [
-    "auto"
-  ]
-
-  locations = [
-    "eu-west-1",
-  ]
-  concurrency = 3
-  api_check_defaults {
-    url = "http://example.com/"
-    headers = {
-      X-Test = "foo"
-    }
-
-    query_parameters = {
-      query = "foo"
-    }
-
-    assertion {
-      source     = "STATUS_CODE"
-      property   = ""
-      comparison = "EQUALS"
-      target     = "200"
-    }
-
-    basic_auth {
-      username = "user"
-      password = "pass"
-    }
-  }
-  environment_variables = {
-    ENVTEST = "Hello world"
-  }
-  double_check              = true
-  use_global_alert_settings = false
-
-  alert_settings {
-    escalation_type = "RUN_BASED"
-
-    run_based_escalation {
-      failed_run_threshold = 1
-    }
-
-    time_based_escalation {
-      minutes_failing_threshold = 5
-    }
-
-    ssl_certificates {
-      enabled         = true
-      alert_threshold = 30
-    }
-
-    reminders {
-      amount   = 2
-      interval = 5
-    }
-  }
-  local_setup_script    = "setup-test"
-  local_teardown_script = "teardown-test"
-}
-```
-
-To add a check to a group, set its `group_id` attribute to the ID of the group. For example:
-
-```terraform
-resource "checkly_check" "test-check1" {
-  name                      = "My test check 1"
-  ...
-  group_id    = checkly_check_group.test-group1.id
-  group_order = 1
-}
-```
-
-The `group_order` attribute specifies in which order the checks will be executed: 1, 2, 3, etc.
-
-
+For documentation and example usage see:
+1. [Checkly's documentation](https://www.checklyhq.com/docs/integrations/terraform/).
+2. [docs/resources](https://github.com/checkly/terraform-provider-checkly/tree/master/docs/resources)
```

Comment:
@ragog I think docs stored in GH there are auto deployed to https://registry.terraform.io/providers/checkly/checkly/latest/docs 
Hence we should link to the registry.
BTW: the docs seem to be broken.
<img width="1095" alt="Screenshot 2020-12-29 at 19 36 12" src="https://user-images.githubusercontent.com/1430734/103306159-2aeae700-4a0d-11eb-9c4c-d423442167c2.png">

Reply:
@hlenke yep, let me change this to a draft and fix the missing points